localisation/00_wc_sad.csv

```diff
@@ -110,7 +110,7 @@ build_cenarion_enclave_desc;Invite your fellow members to create a Cenarion Encl
 raise_host_of_the_wilds;Raise Host of the Wilds;;Versammelt die Schar der Wildnis;;;;;;;;;;;x
 raise_host_of_the_wilds_desc;The Cenarion Circle has many powerful creatures of the forests among its members. Now in times of war you can unleash them upon your enemies.;;Der Zirkel des Cenarius hat viele m�chtige Waldgesch�pfe unter seinen Mitgliedern. Jetzt, in der Zeit des Krieges, k�nnt Ihr sie gegen die Feinde Azeroths einsetzen.;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
-cenarion_circle;Cenarion Circle;;Zirkel des Cenarius;;;;;;;;;;;x
+cenarion_circle;the Cenarion Circle;;Zirkel des Cenarius;;;;;;;;;;;x
```

Comment:
the the

<details>

![image](https://user-images.githubusercontent.com/46576860/64913551-bd00aa80-d74a-11e9-9923-3d763853a04f.png)

</details>

Reply:
Fixed!